grpcbp/tracing.go

```diff
@@ -0,0 +1,40 @@
+package grpcbp
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/reddit/baseplate.go/tracing"
+	"google.golang.org/grpc/metadata"
+)
+
+const ()
+
+func CreateGRPCContextFromSpan(ctx context.Context, span *tracing.Span) context.Context {
+	ctx = metadata.AppendToOutgoingContext(ctx,
+		HeaderTracingTrace, span.TraceID(),
+		HeaderTracingSpan, span.ID(),
+		HeaderTracingFlags, strconv.FormatInt(span.Flags(), 10),
+	)
+	if span.ParentID() != "" {
+		ctx = metadata.AppendToOutgoingContext(ctx,
```

Comment:
style nit: put `ctx,` in its own line as well.

Reply:
nah, having ctx or format strings on the first line is fine

Reply:
well we didn't write an exception for that in the style guide so I'd prefer to either fix the style guide or stick with it.